### ISSUE TITLE
USWDS - Buttons: Increase contrast of Accent Cool hover to increase accessibility.

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds#3491]
## Description

WAVE Browser plugin is flagging contrast issues on the following buttons:
- usa-button--accent-cool (hover)

## Steps to reproduce the issue

1. Get [WAVE plugin](https://wave.webaim.org/extension/)
2. Run the plugin on the [Buttons page](https://designsystem.digital.gov/components/button/)
3. Click `View Details` on Report
![image](https://user-images.githubusercontent.com/3385219/83186554-4e098d00-a0f2-11ea-8258-adb9a24b3f49.png)

4. You should see the following:
![image](https://user-images.githubusercontent.com/3385219/83186497-36ca9f80-a0f2-11ea-8847-6d151bf4ca24.png)

## Additional information

You can also inspect the buttons in firefox click on the text color and see the same error.
![image](https://user-images.githubusercontent.com/3385219/83186860-cb350200-a0f2-11ea-9b35-2f1a76f647a7.png)

![image](https://user-images.githubusercontent.com/3385219/83186835-bf494000-a0f2-11ea-9b79-f42e3c9f37d5.png)

Axe browser plugin only shows the warning for `accent-cool hover` instead of the disabled state. See note below.

---

Note from @pglevy regarding disabled:
>Form elements/buttons that are marked up as disabled are not subject to the usual text contrast guidelines because the state is programmatically understood by assistive tech. See https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html

https://gsa-tts.slack.com/archives/C050HRGN7/p1590695196365400?thread_ts=1590694097.359800&cid=C050HRGN7
